.github/actions/arm64-integration/action.yml

```diff
@@ -0,0 +1,34 @@
+name: ARM64 integration tests
```

Comment:
Is there anything arm specific about this action? Just the path to the CLI?

If this isn't actually reusable, we probably should NOT extract this into an action. For one, we won't get dependabot updates to the actions used here.

Reply:
Ok, this only facilitated testing by calling from a temporary separate workflow. I'll move it back into `release.yaml` if everything else gets a :heavy_check_mark: